sources/splash.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<body>
+  <center>
+    <video id="LoadingAnimation" width="320" height="240" autoplay>
+      <source src="./assets/icons/loading.webm" type="video/webm">
+      Unable to show loading animation
+    </video>
+    <br>
+    <h2>Loading...<h2>

```

Comment:
This doesn't seem to look alright... As font is both `serif` and black by default, this part needs to be properly styled.

I would also like to see it properly localized, and use the one of the strings before I will merge your PR to master.